src/mmw/mmw/settings/base.py

```diff
@@ -130,6 +130,7 @@ def get_env_setting(setting):
 CELERY_CREATE_MISSING_QUEUES = True
 CELERY_CHORD_PROPAGATES = True
 CELERY_CHORD_UNLOCK_MAX_RETRIES = 60
+# CELERYD_CONCURRENCY = 2
```

Comment:
We are currently running 2 Celery workers per machine, with a worker concurrency of 1. See `celery_number_of_workers` and `celery_processes_per_worker` [here](https://github.com/WikiWatershed/model-my-watershed/blob/develop/deployment/ansible/roles/model-my-watershed.celery-worker/templates/celeryd-defaults.j2).

If we plan to tweak these settings, I think it would be good to keep the changes within Ansible so that they can live closer to the process that drives the instance type Celery resides on. Also, tweaking this setting will require monitoring Spark carefully, as the increase in workers can increase the number of things submitted to Spark at a time.